docs/content/en/install/upgrade/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Upgrade notes"
+linkTitle: "Docs"
+no_list: true
+weight: 2
+description: "Notes on upgrading existing installation"
+---
+
+Upgrading the operators is usually a straight-forward operation based on the standard installation method's upgrade procedure. In certain cases however, updates will require certain manual processing to avoid disruptions to the running clusters.
+
+## Updates after operator upgrade to running Cassandra clusters
+
+Sometimes the updates to operators might bring new features or improvements to existing running Cassandra clusters. However, starting from release 1.18 we will no longer update them automatically when the operators are upgraded to prevent a rolling restart on inconvenient time. If there are changes to be applied after upgrading, the ``K8ssandraCluster`` instances are marked with a Status Condition ``RequiresUpdate`` set to True. 
```

Comment:
`on inconvenient time` -> `at inconvenient time`

Reply:
I think the correct form is "at an inconvenient time" ;)

Reply:
"at an inconvenient time" sounds like the best wording to me.